test/helpers/server.ts

```diff
@@ -1,10 +1,12 @@
-/* @flow */
+declare function require (path: string): any
```

Comment:
this declare is not necessary?

Reply:
change to what using `@types/node` to resolve it.